dev/io.openliberty.wsoc.internal_fat/fat/src/io/openliberty/wsoc/endpoints/client/basic/IdleTimeoutTCKClientEP.java

```diff
@@ -100,8 +103,13 @@ public void onClose(Session session, CloseReason closeReason) {
 
     @OnError
     public void onError(Session session, java.lang.Throwable throwable) {
-
-        _wtr.addExceptionAndTerminate("Error during wsoc session", throwable);
+        // Client automatically throws an error when a 1006 response is added. Verify if this
+        // is what is happening on this client and do not throw error when it does since as seen above
+        // on the onClose we expect it to behave this way and need to verify it
+        if (this.EXPECT_TIMEOUT_ERROR && throwable.toString().equals(CLOSE_1006_ERROR_EXCEPTION))
+            LOG.info("Skipping error when receiving a 1006 response since it is expected.");
```

Comment:
Did you encounter this line you local testing? Would you want to log the throwable.toString() message before the if statement?